privacyidea/static/contrib/js/ngmodules/escape-text.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+/**
+ * Wraps the
+ * @param text {string} text which is escaped
+ */
+angular.module('escape_text',[]).filter('escape_text', function () {
+  return function (text) {
+    return text.toString()
+        .replace(/&/g, "&amp;")
+        .replace(/</g, "&lt;")
```

Comment:
Is this sufficient?
What if I pass urlencoded action descriptions like

~~~~
%3Cscript%3Ealert%28%22Got%20you%21%22%29%3C%2Fscript%3E
~~~~